README.md

```diff
@@ -4,6 +4,32 @@
 
 > Font Awesome 5 Ember component using SVG with JS
 
+<!-- toc -->
+
+- [Introduction](#introduction)
+    + [Upgrading Font Awesome?](#upgrading-font-awesome)
+    + [Get started](#get-started)
+    + [Learn about our new SVG implementation](#learn-about-our-new-svg-implementation)
+    + [Going from 0.0.x to 0.1.0](#going-from-00x-to-010)
+- [Installation](#installation)
+- [Add more styles or Pro icons](#add-more-styles-or-pro-icons)
+- [or with Yarn](#or-with-yarn)
```

Comment:
It's odd that these last two aren't sub-headings of the installation section. Seems the headers would need to be adjusted too.

Reply:
Agreed. How does this look https://github.com/scottwernervt/ember-fontawesome/tree/readme-toc?

Reply:
much better! :clap: